pyrometheus/codegen/python.py

```diff
@@ -295,51 +295,63 @@ def get_mixture_internal_energy_mass(self, temperature, mass_fractions):
-        mole_fractions = self.iwts * mass_fractions * self.get_mix_molecular_weight(mass_fractions)
-        viscosities = self.usr_np.sqrt(temperature)*self.get_species_viscosities(temperature)
+        mole_fractions = self.iwts * mass_fractions\
+ * self.get_mix_molecular_weight(mass_fractions)
+        viscosities = self.usr_np.sqrt(temperature)\
+ * self.get_species_viscosities(temperature)
         mix_rule_f = self._pyro_make_array([
             %for sp in range(sol.n_species):
-            ${cgm(ce.wilke_mixture_rule_expr(sol, sp, Variable("mole_fractions"), Variable("viscosities")))},
+            ${cgm(ce.wilke_mixture_rule_expr(sol, sp, Variable("mole_fractions"),
+                                             Variable("viscosities")))},
             %endfor
-            ])   
+            ])
         return sum(mole_fractions*viscosities/mix_rule_f)
 
     def get_species_thermal_conductivities(self, temperature):
         return self._pyro_make_array([
                 % for sp in range(sol.n_species):
-                ${cgm(ce.transport_polynomial_expr(sol.get_thermal_conductivity_polynomial(sp), 1, Variable("temperature")))},
+                ${cgm(ce.transport_polynomial_expr(
+                    sol.get_thermal_conductivity_polynomial(sp), 1,
+                    Variable("temperature")))},
                 % endfor
                 ])
 
     def get_mixture_thermal_conductivity(self, temperature, mass_fractions):
-        mole_fractions = self.iwts * mass_fractions * self.get_mix_molecular_weight(mass_fractions)
-        conductivities = self.usr_np.sqrt(temperature)*self.get_species_thermal_conductivities(temperature)
-        return 0.5*(sum(mole_fractions*conductivities) 
+        mole_fractions = self.iwts * mass_fractions\
+ * self.get_mix_molecular_weight(mass_fractions)
+        conductivities = self.usr_np.sqrt(temperature)\
+ * self.get_species_thermal_conductivities(temperature)
+        return 0.5*(sum(mole_fractions*conductivities)
             + 1/sum(mole_fractions/conductivities))
-                
+
     def get_species_diffusivities(self, temperature):
         return self._pyro_make_array([
-                % for ii in range(sol.n_species):        
+                % for ii in range(sol.n_species):
                 % for jj in range(sol.n_species):
-                ${cgm(ce.transport_polynomial_expr(sol.get_binary_diff_coeffs_polynomial(ii,jj), 1, Variable("temperature")))},
-                % endfor                
-                % endfor  
-                ]).reshape((self.num_species,self.num_species))  
+                ${cgm(ce.transport_polynomial_expr(
+                      sol.get_binary_diff_coeffs_polynomial(ii,jj), 1,
+                      Variable("temperature")))},
+                % endfor
+                % endfor
+                ]).reshape((self.num_species, self.num_species))
 
     def get_mixture_diffusivity(self, temperature, pressure, mass_fractions):
```

Comment:
It is unclear what this does. The name is not descriptive enough. A mixture has thermal, momentum, and mass diffusivities. It appears to return an array of (leading) dimension `num_species`, which implies to me to be the species mass diffusivities. Is this correct? If so, please rename to `get_species_mass_diffusivities`.

Reply:
Thanks for pointing that out. Done.

Reply:
I left a couple of new comments regarding this, but I'll summarize here. We have binary diffusivities (how pairs of species diffuse into each other), and mixture-averaged diffusivities (how one species diffuses into a mixture of all others). It is important that our function names reflect this. Please be sure to use `diffusivities` in function names.

Reply:
I renamed to "get_species_binary_mass_diffusivities" to agree with a comment below as well.